docs/userguide/entry_point.rst

```diff
@@ -486,9 +486,10 @@ entry points remains the same as for console/GUI scripts, and is discussed in th
 .. tip::
     The recommended approach for loading and importing entry points is the
     :mod:`importlib.metadata` module,
-    which is a part of the standard library since Python 3.8. For older versions of
-    Python, its backport :pypi:`importlib_metadata` should be used. While using the
-    backport, the only change that has to be made is to replace ``importlib.metadata``
+    which is a part of the standard library since Python 3.8 and is non-provisional
```

Comment:
I think the issue/reason to use `importlib_metadata` under Python 3.10 is https://github.com/python/importlib_metadata/issues/396 ? Is that something we want to mention?

Reply:
There are other reasons too - basically for early adoption of any behavior that was added to `importlib_metadata` but isn't yet in the supported Pythons. Still, that seems like something for `cpython`/`importlib_metadata` to advertise.